### PR TITLE
dont re-prepare voice messages

### PR DESCRIPTION
**Pull request by kerryarchibald**
Fixes: https://github.com/vector-im/element-web/issues/21206

---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

Preview: https://pr7897--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by DoM1niC** [comment]
Threads Composer is brocken too :(

File: src/audio/Playback.ts [hunk 1]
```diff
@@ -178,7 +181,6 @@ export class Playback extends EventEmitter implements IDestroyable {
             const waveform = Array.from(this.audioBuf.getChannelData(0));
             this.resampledWaveform = makePlaybackWaveform(waveform);
         }
-
```
Comment (turt2live) [inline src/audio/Playback.ts:181]: if possible, it'd be good to keep this blank line for code clarity.

File: src/audio/Playback.ts [hunk 2]
```diff
@@ -133,6 +133,9 @@ export class Playback extends EventEmitter implements IDestroyable {
     }
 
     public async prepare() {
+        if (this.state !== PlaybackState.Decoding) {
+            return;
+        }
```
Comment (turt2live) [inline src/audio/Playback.ts:138]: this needs a comment, and preferably a blank line after it.

**Review by turt2live — CHANGES_REQUESTED** [review]
I don't quite understand what the fix is doing, but it seems like it does fix something. Additional comments in-code to describe what's going on would be appreciated as this area is different from most component structures in the app.

**Review by turt2live — APPROVED** [review]
thanks!

**Pull request merged**